flatpak_build.bash

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
As previously, please remove it. It unnecessarily complicates things that are accomplished with one command.